addons/mail/static/src/messaging/component/composer/composer.xml

```diff
@@ -76,17 +76,20 @@
                                 <div class="o_Composer_toolButtonSeparator"/>
                             </t>
                             <div class="o_Composer_primaryToolButtons">
-                                <EmojisButton
-                                    class="o_Composer_button o_Composer_buttonEmojis o_Composer_toolButton btn-secondary"
-                                    placement="'top'"
-                                    t-att-class="{ 'o-mobile': env.messaging.device.isMobile }"
-                                    t-on-o-emoji-selection="_onEmojiSelection"
-                                    t-ref="emojisButton"
-                                >
-                                    <t t-set="buttonHtml">
+                                <Popover position="'top'" t-on-o-emoji-selection="_onEmojiSelection" t-ref="emojisPopover">
+                                    <!-- TODO FIXME o-open not possible to code due to https://github.com/odoo/owl/issues/693 -->
+                                    <button class="o_Composer_button o_Composer_buttonEmojis o_Composer_toolButton btn btn-secondary"
```

Comment:
not composer component, so should not use composer classes for sharing style

Reply:
But this is `composer.xml`...?

Reply:
you can ignore this comment